skill-tree /src/main/java/com/RDS/skilltree/EndorsementList/EndorsementListService.java

```diff
@@ -0,0 +1,52 @@
+package com.RDS.skilltree.EndorsementList;
+
+import com.RDS.skilltree.Endorsement.EndorsementModel;
+import com.RDS.skilltree.Endorsement.EndorsementRepository;
+import com.RDS.skilltree.Exceptions.NoEntityException;
+import com.RDS.skilltree.User.UserModel;
+import com.RDS.skilltree.User.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class EndorsementListService {
+    @Autowired
```

Comment:
You don't need the `@Autowired` annotation as you're already using a construtor with the required fields.